### DISCUSSION TITLE
Add a comment in nextline automatically when cursor in comment

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#90] [Ideas]
Hi.

What do you think about this feature? Is it compelling?
How hard it is to replicate? 

- https://github.com/doomemacs/doomemacs/blob/master/modules/config/default/config.el#L3

**Comment by abougouffa** [comment]
To be honest, I don't think it worth it. I started MinEmacs after using Doom for some time. I was extremely bothered by Doom's approach of advising every single function to add some minor features like this. At some point, I was facing issues that were practically impossible to debug due to Doom's customizations. I'm really not interested, [as specified in the project's README](https://github.com/abougouffa/minemacs#why), in replicating Doom or Spacemacs, or creating an alternative to them.

**Reply by azzamsa** [reply]
Yes. I think it is the reason I moved away after one year of usage. It breaks at critical times after doom upgrade. Yes, I know it is silly to run `doom upgrade` during those times. But, It is very hard to find where the root cause lies.
